reference/commands/new.rst

```diff
@@ -0,0 +1,78 @@
+.. _reference_commands_new:
+
+conan new
+=========
+
+conan new
+---------
+
+.. code-block:: text
+
+    $ conan new --help
+    usage: conan new [-h] [-v [V]] [--logger] [-d DEFINE] [-f] template
+
+    Create a new recipe (with conanfile.py and other files) from either a predefined or a user-defined template
+
+    positional arguments:
+      template              Template name, either a predefined built-in or a user-
+                            provided one. Available built-in templates: basic,
+                            cmake_lib, cmake_exe, meson_lib, meson_exe,
+                            msbuild_lib, msbuild_exe, bazel_lib, bazel_exe,
+                            autotools_lib, autotools_exe. E.g. 'conan new
+                            cmake_lib -d name=hello -d version=0.1'. You can
+                            define your own templates too by inputting an absolute
+                            path as your template, or a path relative to your
+                            conan home folder.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+      -v [V]                Level of detail of the output. Valid options from less
+                            verbose to more verbose: -vquiet, -verror, -vwarning,
+                            -vnotice, -vstatus, -v or -vverbose, -vv or -vdebug,
+                            -vvv or -vtrace
+      --logger              Show the output with log format, with time, type and
+                            message.
+      -d DEFINE, --define DEFINE
+                            Define a template argument as key=value
+      -f, --force           Overwrite file if it already exists
+
+
+The ``conan new`` command creates a new recipe in the current working directory,
+plus extra example files such as *CMakeLists.txt* or the *test_package* folder (as necessary),
+to be used as a basis for your own project.
+
+Each template has some required and some [optional] user-defined variables used to customize the resulting files. These are:
+
+ * ``basic``: [name], [version], [description], [require, require...]
```

Comment:
can we add some brief descriptions for each of the pre-defined templates? users might not be familiar what is `basic` or `alias` in this context. maybe added some relevant links.
like, what do they do in general, what are their arguments, what files do they generate?